doc/object_finding.rst

```diff
@@ -104,8 +112,17 @@ where one wants to work closer to slit edges:
             maxnumber_sci = 3
             find_trim_edge = 3,3
 
-Finally, if your objects have spectral breaks, like with high-z quasars or
-galaxies, then you can restrict the spectral region that the automated object
+find_min_max
+------------
+
+If your spectrum covers only a minority of the detector 
+(less than 50%), you may need to set this the ``find_min_max``
+parameter.  This includes odd tilts of the grating, 
+data cut off to the blue and objects with 
+spectral breaks, like with high-z quasars or
+galaxies. 
+
+You can restrict the spectral region that the automated object
```

Comment:
Should we make this a wavelength range in future versions? Perhaps an issue on that, what do you think? I think it would not be that hard to implement.

Reply:
Issue #1435 created